handbook/engineering/observability/monitoring_pillars.md

```diff
@@ -177,23 +127,25 @@ We expect `update_commits` to be taking ~50ms and need to confirm that. Is it ~5
 
 ![](https://user-images.githubusercontent.com/3173176/81974154-03f4b780-95da-11ea-8988-069dbdc3ef92.png)
 
-#### What you should do instead
+#### Exceptions
+
+Some graphs will require per-instance granularity. For example, per-container or per-Gitserver replica graphs might be rendered entirely unhelpful if we do not have the ability to observe the behaviour of each individual instance.
+
+#### What you should do
 
 If the information you are presenting is important enough to warrant being shown independently, it should be shown in its own panel. In other words, declare a separate `Query` all-together with a Prometheus selector like `{method="update_commits"}`.
 
 If the information you are presenting is not important enough to warrant being shown independently, or if you're not sure, then aggregate it! It's far better that we have clear panels showing individual numbers and learn we need to add more for specific cases, than it is for us to have every bit of information present such that it is impossible to interpret. We want a low signal-to-noise ratio.
 
 For _instances of services_ in particular, you might be thinking "surely that is an exception?" But no, it is not generally. It is better that an admin knows there is an issue with _a searcher_, or _a gitserver_ and that admin looks through the logs of each to determine the issue than it is to give an information overload which makes it hard to interpret individual panels at all.
 
-### FAQ: Why can't I have all information expanded by default on the dashboard?
-
-This violates the fifth pillar of monitoring at Sourcegraph:
+### Less useful graphs should be hidden by default
```

Comment:
```suggestion
### Complex or less frequently used graphs should be hidden by default
```

Maybe this is a better way to phrase it. I dont know if they are less useful, they might be just too granular, or  otherwise consulted less frequently.

Again, I would not know how to phrase it but I think its basically the "at a glance" vs "deep dive troubleshooting" view.

Reply:
On second thought, what do you think about inverting this pillar?

```suggestion
### Only the most useful and simple graphs should be visible by default
```

Reply:
I like the idea of inverting it, Im not set on the way to phrase it, but LGTM to get started, we can change it later if we find a better way to express it.